docs/compute/live_migration.md

```diff
@@ -169,6 +169,63 @@ spec:
       unsafeMigrationOverride: false
 ```
 
+### Configuring Acceptable Levels of Workload Disruption
+
+The **AllowWorkloadDisruption** parameter enables administrators to define the
+acceptable level of workload disruption during a migration, specifically when it
+fails to converge within the specified `AcceptableCompletionTime`, which is determined by the `completionTimeoutPerGiB` configuration and the VM "size". This option
+provides a balance between ensuring migration completion and mitigating risks
+associated with different migration strategies.
+
+#### Understanding the AllowWorkloadDisruption
+
+The behavior dictated by `AllowWorkloadDisruption` depends on its configuration:
+
+- **When disabled**:
+  - Workloads are not disrupted during migration.
+  - If the migration does not converge within the `AcceptableCompletionTime`, it will be canceled and may need to be restarted.
+  - This approach minimizes workload disruption, however, "busy" workloads may not migrate in time, which increases the risk of repeated migration.
+
+- **When enabled**:
+  - The migration controller is allowed to disrupt the workload associated with long-running migrations to facilitate migration completion.
+  - When `AcceptableCompletionTime` threshold is reached, the migration controller will switch the migration into a post-copy
+    mode. If post-copy is not permitted, the migration controller will instead pause the corresponding Virtual Machine Instance to ensure migration
+    completion.
+
+#### Configuration
+
+To configure `AllowWorkloadDisruption`:
+
+1. **Disabling Workload Disruption**:
+   - Leave `AllowWorkloadDisruption` unset or explicitly set it to `false`.
+   - Example:
+
+     ```yaml
+    spec:
+      configuration:
+        migrations:
+         allowWorkloadDisruption: false
+     ```
```

Comment:
Not sure why, but this block doesn't display properly, same for the next one.

Reply:
yup, no clue :)

Reply:
Looks like it's the indentation. I thought it was the lack of newline but you don't need it if you have exactly 4 spaces of indentation (it currently has 5), which will keep it in line with the numbered list element. 
You could also just have no spaces as the difference isn't particularly noticeable.

Reply:
Ah, yes, now it's there. Thanks @aburdenthehand